kcidb/test_oo.py

```diff
@@ -0,0 +1,101 @@
+"""kcidb.oo test module"""
+
+import os
+import kcidb
+import kcidb.db as db
+import kcidb.oo as oo
+
+from kcidb.test_orm import parse
+
+
+class KCIDB00ClientTestCase(kcidb.unittest.TestCase):
+
+    def _pre_setup(self):
+        self.path = 'sqlite:test_oo.sqlite3'
+        self.source = db.Client(self.path)
+
+        self.initialize_and_load_data()
+        self.assertTrue(isinstance(self.source, kcidb.orm.Source))
+
+    def setUp(self):
+        self._pre_setup()
```

Comment:
You don't really need to create and remove files. Just use SQLite's in-memory database support. I.e. specify the `sqlite::memory:` "database" to `db.Client()`. This way you won't need teardown.